1.8/koding/README.md

```diff
@@ -0,0 +1,58 @@
+# How to use Koding on DC/OS
+
+[Koding](https://wwww.koding.com/) is cloud development platform.
+
+- Estimated time for completion: 10 minutes
+- Target audience: Development teams
+- Scope: Learn how to use Koding on DC/OS.
+
+
+**Table of Contents**:
+
+- [Prerequisites](#prerequisites)
+- [Install Marathon-LB](#install-marathon-lb)
+- [Install Koding](#install-koding)
+- [Access Koding](#access-koding)
+- [Use Koding in production](#use-koding-in-production)
+
+## Prerequisites
+
+- A running DC/OS 1.8 cluster with at least 1 [private agent](https://dcos.io/docs/1.8/overview/concepts/#public) node with 2 CPUs and 1 GB of RAM available.
+- [DC/OS CLI](https://dcos.io/docs/1.8/usage/cli/install/) installed.
+
+## Install Marathon-LB
+
+To make Koding accessible from the public Internet, we will use [Marathon-LB](https://github.com/mesosphere/marathon-lb), a HA-proxy-based load balancer for DC/OS. To install Marathon-LB from the DC/OS CLI, do:
+
+```bash
+$ dcos package install marathon-lb
+We recommend at least 2 CPUs and 1GiB of RAM for each Marathon-LB instance.
+Continue installing? [yes/no] yes
+Installing Marathon app for package [marathon-lb] version [1.4.2]
+Marathon-lb DC/OS Service has been successfully installed!
+See https://github.com/mesosphere/marathon-lb for documentation.
+```
+Note that for advanced Marathon-LB configurations such as enabling HTTPS see the [Marathon-LB Quick Start](https://dcos.io/docs/1.8/usage/service-discovery/marathon-lb/quickstart/) guide.
+
+
+## Install Koding
+
+To install Koding, login to DC/OS and go to the universe tab. Select Koding and and then click on advanced installation. You will be presented with a list of options.
+
+Your first option will be what to name the Koding service. We suggest using the default of "koding".
+
+You will then be able to allocate CPUs and memory to Koding. To run at its best, Koding requires 2 CPUs and 8GB of RAM, although it can run with a minimum of 1GB of RAM.
+
+Next, you will need to pick which node to run Koding on. This is to preserve data across restarts of the Koding service. Select a node with enough available resources (as you just defined) from your available private agent nodes.
+
+After you select a node, you will be asked for a hostname. This hostname is used within the Koding application to define where Koding lives. Individual teams will live at subdomains of this hostname. It mostly exists to be displayed inside the application, but should be where you intend Koding to be accessed from. You may want to make a CNAME record of this host and a wildcard record for any subdomains to point to the FQDN you set up in the next step.
+
+Lastly, to make Koding accessible to the world, we need to use the fully qualified domain name of the public agent node running marathon-lb or the load balancer pointing to your public agent nodes (for instance "DCOS-PublicSlaveLo-1CHPNZSAH97SJ-604928754.eu-west-1.elb.amazonaws.com").
```

Comment:
Node names are written using code, for example `DCOS-PublicSlaveLo-1CHPNZSAH97SJ-604928754.eu-west-1.elb.amazonaws.com`